IPython/kernel/tests/test_message_spec.py

```diff
@@ -187,7 +187,7 @@ class ExecuteInput(Reference):
 
 
 class Stream(Reference):
-    name = Enum((u'stdout', u'stderr'))
+    name = Enum((u'stdout', u'stderr'), allow_none=True)
```

Comment:
I think all of these here should all have `allow_none=False`. I'm guessing that means a default_value must be specified explicitly.